docs/operational_guide/kernel_configuration.md

```diff
@@ -0,0 +1,40 @@
+Kernel Configuration
+====================
+
+This document lists the Kernel tweaks M3DB needs to run well.
+
+## vm.max_map_count
+M3DB uses a lot of mmap-ed files for performance, as a result, you might need to bump `vm.max_map_count`. We suggest setting this value to `262144`, so you don’t have to come back and debug issues later.
+
+On Linux, you can increase the limits by running the following command as root:
+```
+sysctl -w vm.max_map_count=262144
+```
+
+To set this value permanently, update the `vm.max_map_count` setting in `/etc/sysctl.conf`.
```

Comment:
This doc: https://github.com/m3db/m3/blob/cece4eae36f72767fe107b5ce1e015d9ac9548dd/docs/how_to/cluster_hard_way.md

references this file:

https://github.com/m3db/m3/blob/master/integrations/systemd/m3dbnode.service

Do we need to add any of these other configs in there?